### ISSUE TITLE
Inconsistent documentation for thruster config

### ISSUE THREAD

**Issue by binary42** [bluerobotics/bluerobotics.github.io#8]
Inconsistent documentation for thruster config. 

In the build instructions for the Bluerov, the thruster order is not the same as the thruster numbering on the ArduSub documentation site. Thrusters 1,2 and 3,4 are mismatched. This is in regards to the standard bluerov configuration.

**Comment by rjehangir** [comment]
@binary42 Good catch! We changed the numbering scheme in ArduSub recently but have updated those diagrams yet. We'll do that right away.

**Comment by binary42** [comment]
Excellent. I found it when trying to configure the Navio+ with the kit...it had not been without great frustration. I am preparing a write for the forums about it. Very discouraging. I think there is a bug in the navio build of ArduSub. The Navio+ is even flaky with ArduCopter when I test it with a quad. Anyway, that's for later a different post. I've picked up a Pixhawk in any event. Once I get this all straightened out, I think it would be a good idea to create Docker images for the companion computers. I think it will save people days of anger and frustration in setup.
